features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1241,6 +1241,20 @@
         <UseIdentityClaims>{{identity_mgt_account_suspension.use_identity_claims}}</UseIdentityClaims>
      </AccountSuspension>
 
+     <!--This configuration is for enble/disable reCaptcha feature.-->
```

Comment:
Shall we fix the spellings mistake here?

Reply:
Addressed with commit ba9364f549ecbb00ef5f270e4316ff434b660ea0